docs/docs/troubleshooting/TROUBLESHOOTING.md

```diff
@@ -2,75 +2,93 @@
-During the installation, both the installer and Terraform generate some temporary files that are removed after the environment has been uninstalled.
-
-If you installed an environment and then changed any mandatory configuration settings such as `region` or `environment_name`, then clean up 
-all temporary files generated by Terraform before installing a new environment. 
+If you installed an environment and then changed any mandatory configuration settings such as `region` or `environment_name`, then clean up all temporary files generated by Terraform before installing a new environment.
 
 To clean up local Terraform files, run the following command:
-
-```shell
+```
 ./pkg/scripts/cleanup.sh -t
 ```
 
 To clean up locally generated variable files, run the following command:
-
-```shell
+```
 ./pkg/scripts/cleanup.sh -s
 ```
-
 !!! tip
-    You can clean up both the local Terraform files and locally generated variable files by using the `-t` and `-s` switches in one command:
-    
-    ```shell
+    You can clean up both the local Terraform files and locally generated variable files by using the -t and -s switches in one command:
+    ```
     ./pkg/scripts/cleanup.sh -s -t
     ```
 
-!!! info
+!!! info 
     Temporary variable files will be regenerated using the configurating file and overriden during the installation process.
+    
 
-## Interrupted uninstallation
+## Uninstall - Invalid configuration file   
+  
+**Symptom**
+If you try to uninstall an environment by using a different configuration file than the one you used to install it or by using a different version of the code, you may encounter some issues during uninstallation. In most cases, Terraform reports that the resource cannot be removed because it's in use.
 
-### Symptom
+**Solution**
+Idenity the resource and delete it manually from the AWS console, and then restart the uninstallation process.
 
-If you try to uninstall an environment by using a different configuration file than the one you used to install it or by using a different version of the code, you may encounter some issues during uninstallation.
-In most cases, Terraform reports that the resource cannot be removed because it's in use. 
+!!! tip
+    Always use the same configuration file in uninstallation that you used to install the environment. 
+    
+## Uninstall - Stalled product pod
+**Symptom**
+Uninstall fails to remove the persistent volume.
+```
+Error: Persistent volume atlassian-dc-bamboo-share-home-pv still exists (Bound)
 
-### Solution
+Error: context deadline exceeded
 
-Idenity the resource and delete it manually from the AWS console, and then restart the uninstallation process. 
+Error: Persistent volume claim atlassian-dc-bamboo-share-home-pvc still exists with 
+```
+**Solution**
+If a bamboo pod termination stalls, it will block pvc and pv deletion. 
+To fix this problem we need to terminate product pod first and run uninstall command again.
+```
```

Comment:
indicate the code type here using `shell` i.e.

```

  ```shell
  kubectl delete pod <bamboo-pod> -n bamboo --force
  ```